packages/editor/src/components/autosave-monitor/index.js

```diff
@@ -64,13 +64,14 @@ export default compose( [
 			isEditedPostAutosaveable,
 			getReferenceByDistinctEdits,
 			isAutosavingPost,
+			isPostAutosavingLocked,
 		} = select( 'core/editor' );
 
 		const { autosaveInterval } = select( 'core/editor' ).getEditorSettings();
 
 		return {
 			isDirty: isEditedPostDirty(),
-			isAutosaveable: isEditedPostAutosaveable(),
+			isAutosaveable: isEditedPostAutosaveable() && ! isPostAutosavingLocked(),
```

Comment:
What do you think about moving the part which checks whether there is a lock for autosave `! isPostAutosavingLocked()` to the `isEditedPostAutosaveable` selector? It seems like the post can be autosaveable only when there is no lock for autosave.